incubator/tensorflow-inception/values.yaml

```diff
@@ -8,9 +8,10 @@ servicePort: 9090
 containerPort: 9090
 component: "tensorflow-inception"
 replicas: 1
-image: "quay.io/lachie83/inception_serving"
-imageTag: "latest"
-imagePullPolicy: "IfNotPresent"
+image:
+  repository: "quay.io/lachie83/inception_serving"
+  tag: "latest"
```

Comment:
Please use a stable release.

Reply:
What do you mean a stable release? I notice this image only has latest tag. See https://quay.io/repository/lachie83/inception_serving?tag=latest&tab=tags

Reply:
In general, we require stable versions. Is there another image you could use?

@lachie83 Care to provide a stable image?

Reply:
@cheyang I'm currently fixing the GPU support for this chart and updating tensorflow serving. I created the repo with corresponding quay build: https://github.com/thomasjungblut/tfs-inception.

Can I attach the changes in here? I'll also provide a stable tag ;-)

@lachie83 I hope you don't have an issue with replacing your version of the image, couldn't find any dockerfile for it?

Reply:
created https://github.com/kubernetes/charts/pull/6698

Reply:
Sure, I think I can close my fix. Thanks.